src/c++/perf_analyzer/genai-pa/genai_pa/parser.py

```diff
@@ -57,75 +65,116 @@ def add_model_args(parser):
     )
 
 
-def add_profile_args(parser):
+def add_profile_args(parser, exclusive_group):
     parser.add_argument(
         "-b",
-        "--batch-size",
```

Comment:
Why remove the long form of the arg?

Reply:
I wouldn't be too worried about having to do if/else clauses internally in this script - I think one of the goals should be to simplify and abstract away unnecessary PA details or overwhelming numbers of arguments, rather than ease our own forwarding of the arguments to the `perf_analyzer <args>` call internally.

Reply:
If it helps with any of the code, `argparse` also has a `dest` arg that will map the argument name for you. 

ex:
```python
import argparse
parser = argparse.ArgumentParser()

# Converts `args.batch_size` -> `args.b` because of `dest`
# `dest` may also change the help output, so you can also use metavar to manually control that too.
parser.add_argument("-b", "--batch-size", dest="b", metavar="BATCH_SIZE")
args = parser.parse_args()
print(args)
print(args.b)
```

Reply:
metavar and dest were exactly what I needed here. 
Thank you